source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,131 @@ Hardware Checklist
    :local:
    :depth: 2
 
-Stub - detail Hardware Checklist Items
+Use the following checklist when planning the hardware configuration for a production, distributed MinIO deployment.
+
+Considerations
+--------------
+
+When selecting hardware for your MinIO implementation, take into account the following factors:
+
+- Expected amount of data in tebibytes to store at launch
+- Expected growth in size of data for at least the next two years
+- Number of objects by average object size
+- Average retention time of data in years
+- Number of sites to be deployed
+
+Hardware Requirements and Recommendations
+-----------------------------------------
+
+.. note:: 
+
+   For specific hardware suggestions, refer to https://min.io/product/reference-hardware.
```

Comment:
We get a lot of confusion on whether or not we sell hardware, so maybe:

```suggestion
   See our `Reference Hardware <https://min.io/product/reference-hardware?ref=docs` page for a curated selection of servers and storage components from our hardware partners.
   
   MinIO does not provide any hosted services or hardware sales.

```

Reply:
I like it! Added.